index.html

```diff
@@ -732,6 +732,36 @@ <h3>Random Access to Content</h3>
           <li><a href="https://w3c.github.io/wpub/#default-reading-order-manifest">Default Reading Order &gt; Manifest Expression</a></li>
           <li><a href="https://w3c.github.io/wpub/#table-of-contents-infoset">Table of Contents &gt; Infoset Requirements</a></li>
         </ul>
+
+     
+
+  <!--     </section>
+
+       <section id="print-page-reference">
```

Comment:
Narrowed use cases to be more explicitly about page number references. Changed language of the requirement.